frontend/src/metabase/lib/dom.js

```diff
@@ -14,7 +15,16 @@ export const getScrollY = () =>
 // Cypress renders the whole app within an iframe, but we want to exlude it from this check to avoid certain components (like Nav bar) not rendering
 export const isWithinIframe = function () {
   try {
-    return !isCypressActive && window.self !== window.top;
+    if (isCypressActive) {
+      return false;
+    }
+
+    // Mock that we're embedding, so we could visual test embed components
```

Comment:
Since `loki` seems to not run the storybook in iframe, without mocking this we'll get a totally different screen than what we see in normal Storybook stories. This could create confusion for engineers reviewing the PR.

Since many embed components rely on `isWithinIframe`. I think this is the most reliable way to mock embed components. This should have no effect on other Loki snapshots since they're not calling `isWithinIframe`.

### Without this change
#### Storybook
![Screenshot 2024-06-28 at 10 15 11 PM](https://github.com/metabase/metabase/assets/1937582/52004fac-a5ca-4441-aba5-45eef8e16880)

#### Loki snapshot (Notice the footer is appearing in this case)
![chrome_laptop_embed_PublicOrEmbeddedDashboardView_Light_Theme_Default](https://github.com/metabase/metabase/assets/1937582/6652b706-68f3-457b-8a96-6e34bb9d2ac6)

Reply:
Agree with Denis on that